### DISCUSSION TITLE
any configuration to change log level or enable/disable logging ?

### DISCUSSION THREAD

**Discussion by gasuketsu** [wezterm/wezterm#1749] [Q&A] [answered]
I'm using wezterm on Windows environment and found that log files "gui-log-xxxxx.txt"  are saved under `$USERPROFILE\.local\share\wezterm` directory.
It looks that gui-log contains INFO or higher level logs. Is there any option to change log level or enable/disable logging to reduce/increase logs ?

**Comment by wez** [comment, marked as answer]
You can set the environment variable `WEZTERM_LOG` before launching wezterm to change the log level. setting it to `debug` will provide fairly verbose logging.

**Reply by gasuketsu** [reply]
Thanks for quick response!
